docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc

```diff
@@ -1,15 +1,17 @@
 [[prebuilt-rules-changelog]]
 == Prebuilt rule changes per release
 
-[float]
-=== 7.12.0
+beta[]
 
+The following lists prebuilt rule updates per release. Only rules with
+significant modifications to their query or scope are listed. For detailed
+information about a rule's changes, see the rule's description page.
 
-This prebuilt rule has been deprecated:
-
-* setgid-bit-set-via-chmod
+[float]
+=== 7.12.1
 
-These prebuilt rules have been updated:
+[float]
+=== 7.12.0
```

Comment:
Should we add a `N/A` here if there are no major updates? Looks a little odd with the standalone heading.

Reply:
oh, to `7.12.1` good catch. Or maybe `formatting changes only`